src/Cli/dotnet/Installer/Windows/InstallRequestMessage.cs

```diff
@@ -120,6 +120,14 @@ public string WorkloadId
             set;
         }
 
+        /// <summary>
+        /// The new mode to use: workloadset or loosemanifests
+        /// </summary>
+        public bool WorkloadMode
```

Comment:
Naming nit: Calling a boolean property `WorkloadMode` doesn't tell you what true or false values mean.  I think it would be better as `UseWorkloadSets`, `WorkloadSetsEnabled`, or similar.  This also applies to method parameters where this value is passed, etc.

```suggestion
        public bool UseWorkloadSets
```